### PR TITLE
improved overall look of readme and remove unused import axios.

### PR DESCRIPTION
**Pull request by DebiprasadXD**

### CODE REVIEW

**Comment by roj1512** [comment]
WoW

**Comment by DebiprasadXD** [comment]
> WoW

not a major change lmao

**Comment by roj1512** [comment]
It's still good to be honest. People should take care of cases, even. [Just like I do.](https://github.com/nim-lang/Nim/commit/6b302bb71b6d9aab708b91f8d68957652ec30480).

File: README.md [hunk 1]
```diff
@@ -2,19 +2,23 @@
 
 [![Deploy to Heroku](https://github.com/ArnabXD/TGVCBot/actions/workflows/heroku.yml/badge.svg?branch=main)](https://github.com/ArnabXD/TGVCBot/actions/workflows/heroku.yml)
 [![TypeScript Build](https://github.com/ArnabXD/TGVCBot/actions/workflows/typescript.yml/badge.svg?branch=main)](https://github.com/ArnabXD/TGVCBot/actions/workflows/typescript.yml)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://GitHub.com/ArnabXD/TGVCBot/graphs/commit-activity)
```
Comment (ArnabXD) [inline README.md:5]:
```suggestion

[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/ArnabXD/TGVCBot/graphs/commit-activity)
```

File: README.md [hunk 2]
```diff
@@ -27,17 +31,17 @@
 | THUMBNAIL    | Direct link for default thumbnail (used if audio has no thumbnails).                   | False    |
 | WATERMARK    | Text to render on bottom left corner of Banner. Default: `TGVCBot`. (10-14 chars)      | False    |
 
-### Tutorial
+### Tutorial 🔥: 
 
 - VPS/Local : [Tutorial](https://blog.arnabxd.me/deploy-tgvcbot-in-a-ubuntu-vps)
 
-### Memes
+### Memes🤣 :
```
Comment (ArnabXD) [inline README.md:38]:
```suggestion
### Memes 🤣: 
```

**Review by ArnabXD — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -48,11 +52,20 @@
...
+**the project? Consider sponsoring.**
+
+[![ForTheBadge built-with-Typescript](http://ForTheBadge.com/images/badges/built-with-love.svg)](https://GitHub.com/ArnabXD/TGVCBot)
```
[the remark this answers is not in the document]
Comment (DebiprasadXD) [reply README.md:70]: okay I get it 😄

**Review by ArnabXD — APPROVED** [review]

**Pull request merged**